source/game.production.script.bmx

```diff
@@ -533,8 +537,26 @@ Type TScript Extends TScriptBase {_exposeToLua="selected"}
 
 			If script.subScripts
 				'#440 propagate final optional header flags to episodes
```

Comment:
#440 - kann die Referenz raus oder war die drin um herausfinden zu koennen, warum folgendes gemacht wird?

Reply:
Die Referenz war ja vorher schon drin, ich würde sie hier also nicht entfernen wollen.
Ich finde es für das Tracing eigentlich immer ganz hilfreich, Ticket-Referenzen zu haben. Wenn sie weder im Code stehen noch in den Commitnachrichten, wird es schwer, herauszufinden, welches Ticket/welcher PR dahinterstand.

Reply:
Jein ... dafuer hast Du ja "blame", was dir sagt, wann welche Codezeile angefasst wurde.

Aber du hast Recht, dass wir die Zeile hier nicht anfassen muessen. Und "schaden" tut sie eh nicht.